### PR TITLE
Room::ReverseLocalPosition

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

**Comment by Misaka-ZeroTwo** [comment]
`WorldPosFromLocalPos`, `PositionFromLocalPos`. Aren't great name but I don't feel ReverseLocalPosition is a intuitive name.

**Comment by louis1706** [comment]
yeah i was not knowing what name to give for it

File: Exiled.API/Features/Room.cs [hunk 1]
```diff
@@ -291,6 +291,13 @@ public static Room Random(ZoneType zoneType = ZoneType.Unspecified)
         /// <returns>Local position, based on the room.</returns>
         public Vector3 LocalPosition(Vector3 position) => Transform.TransformPoint(position);
 
+        /// <summary>
+        /// Returns the World position position, based on a local space position.
```
Comment (FakeMan2332) [inline Exiled.API/Features/Room.cs:295]: position postion

Comment (FakeMan2332) [inline Exiled.API/Features/Room.cs:295]:
```suggestion
        /// Returns the World position, based on a local space position.
```

File: Exiled.API/Features/Room.cs [hunk 2]
```diff
@@ -276,6 +276,13 @@ public static Room Random(ZoneType zoneType = ZoneType.Unspecified)
         /// <returns>Local position, based on the room.</returns>
         public Vector3 LocalPosition(Vector3 position) => Transform.TransformPoint(position);
 
+        /// <summary>
+        /// Returns the World position, based on a local space position.
+        /// </summary>
+        /// <param name="offset">Local position.</param>
+        /// <returns>World position, based on the room.</returns>
+        public Vector3 ReverseLocalPosition(Vector3 offset) => Transform.InverseTransformPoint(offset);
```
Comment (NaoUnderscore) [inline Exiled.API/Features/Room.cs:284]: Just name it `WorldPosition`

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]

**Pull request merged**